hack/deploy-hub-local/build-edgecluster.sh

```diff
@@ -45,8 +45,20 @@ export OC_ACM_VERSION="${acm_version}"
 export OC_ODF_VERSION="${odf_version}"
 export HUB_ARCHITECTURE="${5:-compact}"
 export SINGLE_NIC="${6:-true}"
+export CLUSTER_NAME=${CLUSTER_NAME:-edgecluster}
```

Comment:
overlapping var name with the Makefile

Reply:
mmh, not sure I follow. Are you referring at the name being the same as in the `Makefile` ? if it's that, then I did it on purpose so that we can set the variable when running `make`. 

Or

Are you referring to the default being defined in both places? We need it in both as the `Makefile` uses the clsutername for the `edgecluster` file name

Reply:
yes, i think we should not use overlapping VAR incase we want to use them else where - also make sure that we are not overwriting stuff by mistake ( an issue when using bash )

maybe using something like this 
```suggestion
export EDGE_CLUSTER_NAME=${CLUSTER_NAME:-edgecluster}
```

Reply:
mmh! Can you elaborate more on why this is a problem? 

With your suggestion, we are still taking the default value from `CLUSTER_NAME`, which in practice makes the `Makefile` definition overwrite the bash value. 

Just trying to understand :)

Reply:
so i think that it can be confusing using the same Variable name and different places /file or module ( in bash?! )
also when you use the same one you can overwrite the value and have some major issues later on. 

i think it best to avoid it if we can - on other languages it not a big deal cause the value is not deleted